### ISSUE TITLE
[BUG] developer organization name on iOS App Store still shows Loki Project despite of rebranding

### ISSUE THREAD

**Issue by fracting** [oxen-io/session-ios#842]
### Code of conduct

- [X] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-ios/blob/master/CODE_OF_CONDUCT.md)

### Self-training on how to write a bug report

- [X] I have learned [how to write a bug report](https://www.browserstack.com/guide/how-to-write-a-bug-report)

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

iOS App Store shows "the Loki Project" as the developer of Session iOS and Oxen iOS.

https://apps.apple.com/us/developer/the-loki-project/id1465686521

<img width="522" alt="App Store" src="https://user-images.githubusercontent.com/194342/232983205-e53e8386-4064-449a-b18c-15fe039f61c3.png">

### Expected Behavior

The organization should be "The Oxen Project" instead, like what Android Google Play shows:

https://play.google.com/store/apps/developer?id=Oxen+Project

<img width="1420" alt="Google Play" src="https://user-images.githubusercontent.com/194342/232983888-60148402-72ac-4a1e-93ad-cf29c5a10e8c.png">

### Anything else?

ChatGPT:
 
> Yes, it's possible to change your developer name on the App Store, but the process may vary depending on whether you are an individual or part of an organization.
> 
> If you are an individual developer:
> Unfortunately, individual developers cannot change their developer name displayed on the App Store directly, as it is based on the first and last name provided during enrollment in the Apple Developer Program. However, you may consider contacting Apple Developer Support to see if they can assist you with this issue.
> 
> If you are part of an organization:
> 
> Sign in to App Store Connect (https://appstoreconnect.apple.com/).
> Click on "Users and Access."
> Under the "Legal" section, click on the "Edit" button next to "Organization Name."
> Update your organization name and click "Save."
> Keep in mind that changing the organization name will also change the developer name displayed on the App Store for all apps under that organization.
> 
> Please note that updating the organization name in App Store Connect doesn't automatically update the name in the Apple Developer Program. To update the organization name there, you'll need to contact Apple Developer Support (https://developer.apple.com/contact/). Additionally, you might need to provide updated legal documentation to reflect the new organization name.
> 
> It's important to consider the potential impact of changing your developer name, as users might have a hard time recognizing your apps under the new name. Be prepared to communicate the change to your users if needed.

> Changing your organization name will not affect the existing user reviews for your apps. The reviews and ratings are tied to the apps themselves, not the developer or organization name. When you change the organization name, the reviews and ratings will still be associated with the respective apps on the App Store. However, keep in mind that users may be confused if they notice a different developer name associated with the app, so it's a good idea to communicate any changes to your user base if necessary.

I didn't verify if ChatGPT is correct.

**Comment by KeeJef** [comment]
Yeah we are aware of this one, unfortunately we have contacted Apple and they wont change the name of our organisation, because the account is not registered under a legal entity which bears the Oxen or OPTF name currently, the solution seems to require registering a new Apple developer account under the OPTF and then transferring app ownership to the new account, which is quite burdensome considering the migrations which would have to occur. Nonetheless we are pursuing this idea, its just taking much longer than expected.
